cu-boulder/vivo-template
# change for 5726 - class exclusions from search summary

VIVO currently shows classes like "entity" "occurrant" "person" and other classes in the summary lines for all returned results. This is not useful or desirable.
Example from a search for Geology people on VIVO shows:
Carignan, Kelly S Professional Research Asst
... Agent Continuant Entity Faculty Member Independent Continuant Person Coastal Geology Physical Geography Professional Research Asst Digital elevation model ...
The VIVO community created a fix for the develop branch, which is 1.11, work with them to apply this to version 1.9

This applies to VIVO Jira 1639
Pull request ​https://github.com/vivo-project/Vitro/pull/110 was approved and merged into the maint-rel-1.9 branch of Vitro.

To verify this template is applied 
Do: git log in the Vitro branch you should see the following commit:
commit e9f29c9ab0693de3db5c80de4c4be79ad08cb9e4
Author: Don Elsborg <elsborg@colorado.edu>
Date:   Tue Jan 8 14:29:42 2019 -0700

    applied class exclusion from search results to 1.9 release

Once this is approved and applied to production wush needs to be updated at:
https://wush.net/trac/fis/wiki/CuExperts#CurrentVIVOrelease 
with the latest commit hash for master
